compute/vm-image-spec-bookworm.yaml

```diff
@@ -40,7 +40,7 @@ commands:
     sysvInitAction: respawn
     shell: '/bin/sql_exporter -config.file=/etc/sql_exporter_autoscaling.yml -web.listen-address=:9499'
   - name: rsyslogd
-    user: postgres
+    user: root
```

Comment:
@lubennikovaav will this change work with pgaudit enabled?

I need to run rsyslog as root so that it could create unix pipes under `/dev/log`. Needed for `imuxsock` module.

Reply:
can we just change the permissions of the directory to make it available for postgres user?
see code in this file below

Have you tested if compute_ctl is able to restart (pkill) rsyslog that was started by root?
I remember that I had permission issues with this setup.

Reply:
Indeed, postgres user cannot kill rsyslog started by root. Discussing the options here: https://neondb.slack.com/archives/C03TN5G758R/p1741776636293209

Reply:
Updated to keep running rsyslog under postgres user.